src/main/java/tech/code0/communication/RabbitConnection.java

```diff
@@ -0,0 +1,42 @@
+package tech.code0.communication;
+
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import lombok.Getter;
+import tech.code0.configuration.AquilaConfiguration;
+
+import static tech.code0.util.AquilaLogger.LOGGER;
+
+@Getter
+public class RabbitConnection {
+
+    private final Connection connection;
+
+    public RabbitConnection(AquilaConfiguration aquilaConfiguration) {
+        LOGGER.info("Initializing Rabbit connection");
+        this.connection = createConnection(aquilaConfiguration);
+    }
+
+    private Connection createConnection(AquilaConfiguration aquilaConfiguration) {
+        final var connectionFactory = new ConnectionFactory();
+        connectionFactory.setHost(aquilaConfiguration.getRabbitMQHost());
+        connectionFactory.setPort(aquilaConfiguration.getRabbitMQPort());
+
+        try (final var connection = connectionFactory.newConnection()) {
+            LOGGER.info("Connected to RabbitMQ");
+            return connection;
+        } catch (Exception exception) {
+            LOGGER.warning("Connection to RabbitMQ failed: " + exception.getMessage());
+            throw new RuntimeException(exception);
+        }
+    }
+
+    public void close() {
+        try {
+            LOGGER.warning("Closing RabbitMQ connection");
```

Comment:
Closing connections is a standard procedure during an application lifecycle, so this is not really a warning
```suggestion
            LOGGER.info("Closing RabbitMQ connection");
```